src/renderer/components/dialogs/ForwardMessage/styles.module.scss

```diff
@@ -0,0 +1,4 @@
+// We need to force the dialog to full height to make `AutoSizer` work
+.forwardMessageDialog {
+  height: 100%;
+}
```

Comment:
Every-time we use `AutoSizer` in a dialogue a hack is required, I wonder if this can be improved